[MIG] public_budget: Migration to 15.0

pablohmontenegro opened the pull request.

pablohmontenegro committed:
[MIG] public_budget: Migration to 15.0

roboadhoc commented:
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/odoo-public-administration/pull/437).

Commit:
[REM] deprecate check functionality
Borramos distintas funcionalidades de cheques que cambiaron mucho con el nuevo release. Las borramos porque en 2022 no están usando cheques
